gbus/worker.go

```diff
@@ -346,11 +347,22 @@ func (worker *worker) processMessage(delivery amqp.Delivery, isRPCreply bool) {
 	bm, err := worker.extractBusMessage(delivery, msgSpecificLogEntry)
 	if err != nil {
 		worker.span.LogFields(slog.Error(err), slog.String("grabbit", "message is poison"))
-		//reject poison message
+		// reject poison message
 		_ = worker.reject(false, delivery, msgSpecificLogEntry)
 		return
 	}
-
+	isDuplicate, err := worker.handleDuplicates(bm, delivery, msgSpecificLogEntry)
```

Comment:
wouldnt you want to "handle duplicates" in case of global handlers/ dead letter handlers invocation as well?

Reply:
the deduplication is on the GbusMessage level in dlq/global we don't yet have the gbusMessage yet